.circleci/config.yml

```diff
@@ -1105,11 +1105,11 @@ jobs:
   # Builds in C++20 mode and uses debug build in order to speed up.
   # Do *NOT* store any artifacts or workspace as we don't run tests on this build.
   b_ubu_cxx20:
-    <<: *base_ubuntu2404_large
+    <<: *base_ubuntu2404_xlarge
     environment:
-      <<: *base_ubuntu2404_large_env
+      <<: *base_ubuntu2404_xlarge_env
       CMAKE_BUILD_TYPE: Debug
-      CMAKE_OPTIONS: -DCMAKE_CXX_STANDARD=20
+      CMAKE_OPTIONS: -DCMAKE_CXX_STANDARD=20 -DCMAKE_EXE_LINKER_FLAGS=-fuse-ld=gold
```

Comment:
It _is_ a bit eerie that we just removed that and everything worked and the next thing touched needs it back :-D...

Reply:
It doesn't actually need it back - I added it because @clonker asked for it.

Reply:
It could have prevented going to the xlarge configuration.